flang/lib/Optimizer/CodeGen/CodeGen.cpp

```diff
@@ -20,13 +20,10 @@
 #include "flang/Optimizer/Dialect/FIRAttr.h"
 #include "flang/Optimizer/Dialect/FIRDialect.h"
 #include "flang/Optimizer/Dialect/FIROps.h"
-#include "flang/Optimizer/Dialect/FIRType.h"
-#include "flang/Optimizer/Support/FIRContext.h"
 #include "flang/Optimizer/Support/InternalNames.h"
 #include "flang/Optimizer/Support/KindMapping.h"
 #include "flang/Optimizer/Support/TypeCode.h"
 #include "mlir/Conversion/ArithmeticToLLVM/ArithmeticToLLVM.h"
-#include "mlir/Conversion/LLVMCommon/TypeConverter.h"
```

Comment:
It's not clear why these are being removed. Are they no longer needed? If not, why not?

Reply:
I think they have been moved to the `TypeConverter.h`

Reply:
This change is from: https://reviews.llvm.org/rGf1a2b507896c3fec8b725e489d9c0a9816d292c0

Reply:
Let me expand - these headers were missing from `TypeConverter.h` (so that `TypeConverter.h` includes everything that it needs). From what we could tell, these are not needed in `CodeGen.cpp`.